NAMESPACE

```diff
@@ -2,8 +2,6 @@
 
 S3method(print,CellValue)
 S3method(print,RowsVerticalSection)
-export("[")
-export("[<-")
```

Comment:
these where previously reexported from base r @gmbecker

Reply:
we don't need them anymore? @Melkiades @gmbecker

Reply:
These were base r subsetting reexported. It was not strictly necessary but it was a good solution to the older problem of the reference page